aws/data_source_aws_lambda_function.go

```diff
@@ -40,6 +40,22 @@ func dataSourceAwsLambdaFunction() *schema.Resource {
 					},
 				},
 			},
+			"file_system_config": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"arn": {
+							Type:     schema.TypeString,
+							Required: true,
```

Comment:
Please add `Computed: true,` as this attribute is "output-only".
Remove `Required`.

Reply:
Updated.